File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/user/dto/ResetPasswordRequest.java
package com.rolandvitezhu.todocloud.network.api.user.dto;

import com.google.gson.annotations.SerializedName;

public class ResetPasswordRequest {

  @SerializedName("email")
  public String email;

  public String getEmail() {
    return email;
  }

  public void setEmail(String email) {
    this.email = email;
  }
}

File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/user/dto/ResetPasswordRequest.kt
package com.rolandvitezhu.todocloud.network.api.user.dto

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class ResetPasswordRequest (
        @SerializedName("email")
        var email: String?
) : Parcelable {
    constructor() : this(
            null
    )
}